Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -41,7 +38,7 @@ public QuestionRepository(TrappistDbContext dbContext, UserManager<ApplicationUs
         {
             var question = Mapper.Map<QuestionDetailAC, Question>(questionAC.Question);
             var singleMultipleAnswerQuestion = Mapper.Map<SingleMultipleAnswerQuestionAC, SingleMultipleAnswerQuestion>(questionAC.SingleMultipleAnswerQuestion);
-            question.ApplicationUser = await _userManager.FindByEmailAsync(userEmail);
+            //To-Do Add created by user Id 
```

Comment:
He has to add
`question.CreatedByUserId = userId;`
instead of 
`question.ApplicationUser = await _userManager.FindByEmailAsync(userEmail);`